tokens/movistar-classic.json

```diff
@@ -1478,6 +1478,38 @@
       "type": "borderRadius"
     }
   },
+  "text": {
+    "weights": {
+      "cardTitle": {
+        "value": "bold",
+        "type": "typography"
+      },
+      "text-preset-5": {
```

Comment:
In order to avoid having text twice, maybe we could name them only as preset-n?

Reply:
I'm sure that has to be redefined for native Figma Tokens. Now is the name that appears in Figma Tokens plugin
![image](https://user-images.githubusercontent.com/6722153/234351593-5490675c-afb1-448e-8203-333fa62cf9ba.png)